ansible_collections/arista/avd/molecule/eos_designs_deprecated_vars/inventory/host_vars/port-channel-short-esi.yml

```diff
@@ -0,0 +1,22 @@
+type: l2leaf
+l2leaf:
+  nodes:
+    - name: port-channel-short-esi
+
+servers:
+  - name: server01_ESI
+    rack: RackC
+    adapters:
```

Comment:
Add another adapter with `short_esi` _and_ `ethernet_segment.short_esi` set, to still test that `ethernet_segment.short_esi` overwrites the other one. (Those tests got removed from the regular molecule when you removed the deprecated variable)

Reply:
Added.